assets/src/css/admin/give-admin-global.scss

```diff
@@ -0,0 +1,11 @@
+/**
+ * Give Admin SCSS/CSS
+ *
+ * @package:     Give
+ * @subpackage:  SCSS/Admin
+ * @copyright:   Copyright (c) 2016, GiveWP
+ * @license:     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+*/
+
+// Stripe Core
+@import 'stripe-admin';
```

Comment:
@DevinWalker We can do micro-optimization here: CSS which we are outputting globally on the admin page, already defined in Stripe. To take leverage of Give core, we can remove duplicate CSS from Stripe Premium: https://github.com/impress-org/give-stripe/blob/develop/assets/src/css/admin/settings.scss


![image](https://user-images.githubusercontent.com/1784821/69619911-683eb180-1062-11ea-8049-1a5d9b5d4872.png)